### PR TITLE
Color change when hovering over hamburger menu

### PR DESCRIPTION
**Pull request by DominicRoyStang**
Added shading on hover for the floating action button (hamburger menu toggle).
This makes the floating action button consistent with other pink buttons on the website.

Quick demo after applying these changes in the inspector:

_Let me know if you have any questions._

### CODE REVIEW

**Comment by jackalyst** [comment]
Thank you for yet another PR! For consistency with shadow's, there's a variable $shadow which can be found in _variables.scss

`0px 3px 10px rgba(0, 0, 0, 0.3)`

It might be to subtle for this mind you.

**Comment by DominicRoyStang** [comment]
**TL;DR**: I'm not exactly sure if you're talking about the on hover color or the button's box-shadow (or other). If you mean the hover color, I'm going to need more details. If you're talking about the shadows added in pull request https://github.com/theQRL/theqrl.org/pull/16 , I'm okay with using $shadow, or going full material-design style.


If you're talking about the color change on hover, I based it off the code for `a.cta:hover` in `_custom.scss`.

```
a.cta {
	background-color:#dc255d;
	padding:10px 25px;
	box-shadow: $shadow;
	transition: background-color 0.3s;
	/* ... */
	&:hover {
		background-color:$quantum_hover;
	}
}
```
_Side Note: Speaking of which, I should probably use the parent selector SCSS notation like in the code above for consistency. I'll update this for the next change to this pull request._

best I can do to use $shadow is setting
```
#nav-icon:hover {
    box-shadow: inset $shadow;
}
```
which would look like this (with shadow being set to $shadow for the default state).


In my opinion, this looks weird for a floating action button (FAB), so I'm assuming you're talking about the shadows added in pull request https://github.com/theQRL/theqrl.org/pull/16

In that case, I just didn't feel that the other shading was sufficient for a FAB, but that's really mostly personal preference, so I don't mind changing it to $shadow for consistency (see gif above before pre-hover to see what that looks like).

Technically, material design guidelines also agree that a FAB [should be higher](https://material.io/guidelines/material-design/elevation-shadows.html#elevation-shadows-shadows) than a regular button, but they are guidelines, not rules, so I'm not too picky about this.

Material guidelines also state that the button should have a [more pronounced shadow](https://material.io/guidelines/components/buttons.html#buttons-raised-buttons) when clicked, so let me know if you'd like me to fully implement the FAB with material design style for the #nav-icon id.

**Comment by jackalyst** [comment]
Sorry I was late in getting back to you, I went back to our designer to Discuss this and how far into material design we were going (or if we were going for material design in the first place - because it does seem to borrow some elements from it).

Strictly speaking, The QRL design is not material design, as much as it may appear so.

In that way, we're probably going to go back and lessen the shadow to what is defined elsewhere (or not at all), the colour change on hover is staying.

It's possible that in the future we're going to go towards more of a material design route (or some other route), but we'll need to revisit it down the road with regards to everything in our platform.

As an aside, I really enjoyed the way you composed your conversation with added gif's to boot!
